LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [2020] [Joseph Guillaume; Felix Andrews]
```

Comment:
```suggestion
Copyright (c) 2020, Joseph Guillaume and Felix Andrews.
```

This would be more conventional punctuation?

Reply:
License is actually GPL>=2 https://github.com/josephguillaume/hydromad/blob/master/DESCRIPTION#L17
Github also has template licenses that can be applied: https://help.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository
I can add that if this PR removes this file?

Reply:
I can also change the license.txt? What do you prefer? I used the suggested github link, which where I came out to the MIT, but I didn't check your description :-(

Reply:
I have now removed LICENSE.txt and pointed to DESCRIPTION